test/remote/gateways/remote_mercado_pago_test.rb

```diff
@@ -90,6 +92,21 @@ def test_successful_purchase_with_american_express
     assert_equal 'accredited', response.message
   end
 
+  def test_successful_purchase_with_taxes_and_net_amount
+    # Minimum transaction amount is 0.30 EUR or ~1112 $COL on 1/27/20.
+    # This value must exceed that
+    amount = 10000_00
+
+    # These values need to be represented as dollars, so divide them by 100
```

Comment:
Instead of taking the extra action to divide by 100 here, what about changing the amount to `10000`?

Reply:
Since this is a remote test, it has to abide by the logic `transaction_amount = net_amount + taxes` or the transaction will fail. This is a bit confusing but amount is in cents because it is a supported ActiveMerchant field and the `net_amount` and `taxes` are in dollars because we just pass them through and that is what the gateway accepts.

Reply:
Ah, that makes sense! Thanks for explaining!